scripts/updatePlinkCode.sh

```diff
@@ -0,0 +1,17 @@
+# update the local repo with local copies of the plink-ng 
```

Comment:
These scripts to do this automatically are a good idea.  Ideally we'll be able to use the library directly at some point, but this at least makes it somewhat less insane.  

As part of this operation I might have it automatically populate a README or something in the same folder with the git tag of the version it's copying in so that that information is recorded somewhere and not just in the commit comments.

Reply:
Yeah, I'll do that - it sounds more reliable than the commit comment.